pkg/distributors/test/RewardsScheduler.test.ts

```diff
@@ -0,0 +1,143 @@
+import { ethers } from 'hardhat';
+import { Contract, BigNumber } from 'ethers';
+import { expect } from 'chai';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/dist/src/signer-with-address';
+
+import Token from '@balancer-labs/v2-helpers/src/models/tokens/Token';
+import TokenList from '@balancer-labs/v2-helpers/src/models/tokens/TokenList';
+
+import { fp } from '@balancer-labs/v2-helpers/src/numbers';
+
+import { deployedAt } from '@balancer-labs/v2-helpers/src/contract';
+import { expectBalanceChange } from '@balancer-labs/v2-helpers/src/test/tokenBalance';
+import * as expectEvent from '@balancer-labs/v2-helpers/src/test/expectEvent';
+import { advanceTime, currentTimestamp } from '@balancer-labs/v2-helpers/src/time';
+import { setup, rewardsDuration } from './MultiRewardsSharedSetup';
+
+describe('Rewards Scheduler', () => {
+  let admin: SignerWithAddress, lp: SignerWithAddress, rewarder: SignerWithAddress;
+
+  let rewardTokens: TokenList;
+  let vault: Contract;
+  let stakingContract: Contract;
+  let rewardsScheduler: Contract;
+  let rewardsToken: Token;
+  let pool: Contract;
+
+  before('deploy base contracts', async () => {
+    [admin, , lp, , rewarder] = await ethers.getSigners();
+  });
+
+  sharedBeforeEach('set up asset manager and mock callback', async () => {
+    const { contracts } = await setup();
```

Comment:
If we're going to rely so heavily on `setup`, it should export the signers as well instead of having us keep both calls in sync.

Reply:
Makes a lot of sense - I updated the RewardScheduler test to use values exported from setup and will update the other tests to use them too in a separate PR